app/src/components/main/modal/modalLayout/modalFooter/modalFooter.jsx

```diff
@@ -100,7 +98,7 @@ export class ModalFooter extends Component {
             <div className={cx('button-container')}>
               <BigButton
                 color={okButton.danger ? 'tomato' : 'booger'}
-                onClick={onClickOk}
+                onClick={() => okButton.onClick(onClickOk)}
```

Comment:
`onClick={() => okButton.onClick(closeHandler)}`

Reply:
For now `closeHandler` run with confirmation logic. For `okButton` we don't need to do it.